docs/getting-started/install/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Install Shopify CLI
+section: getting-started
+toc: false
+redirect_from: "/install/"
+---
+
+Shopify CLI can be installed using a variety of package managers.
+
+### Ruby gem
+
+```console
+$ gem install shopify-cli
+```
+
+### apt (Debian, Ubuntu)
+
+Currently, you’ll need to install a downloaded .deb file with an explicit version number. Make sure you check the [release page](#) to make sure you install the latest package.
```

Comment:
The "currently" seems to suggest that this will change. Can we say what the future plan is? If not, maybe leave that out (here and below)?

Reply:
@mkevinosullivan I thought that in future they'd be able to just call `apt install shopify-cli` without having to specify a version or a .deb file — did I misunderstand?

Reply:
Possibly - we haven't looked into hosting the `.deb` or `.rpm` files yet since these two files are small and are basically wrappers around `gem install`.  We may not decide to go that path, but it would be the nicest experience.

I would remove the "Currently" and we'll tweak the copy if/when we decide to host the package files such that `apt install shopify-cli` would work.

Reply:
Got it. Zapped!